api/mediasdk_structures/ts_struct_decl.h

```diff
@@ -890,6 +890,11 @@ STRUCT(mfxExtCodingOption3,
     FIELD_T(mfxU16, EncodedUnitsInfo)
     FIELD_T(mfxU16, EnableNalUnitType)
     FIELD_T(mfxU16, ExtBrcAdaptiveLTR)     /* tri-state option for ExtBrcAdaptiveLTR */
+    FIELD_T(mfxU32, RepackMaxIFrameSize)
```

Comment:
version check missed entirely

Reply:
The version check is define in line 838.

Reply:
You define new fields in the primary mediasdk uAPI. You need another version check with MFX_VERSION_NEXT.

Reply:
Yes, it is. Please find it in line 838: #if (MFX_VERSION >= MFX_VERSION_NEXT)

Reply:
Hm. Yes, indeed. But this looks to be a bug. @onabiull : this definition in ts_struct_decl.h don't correspond to uAPI definition...